docker/prod/environment

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
I think I mentioned this before, but this file isn't actually needed, since the environment variables can just be set on the container (in the docker-server-configs repo). And that's better than hardcoding them in the image: if we move the container (or move consul) we'd have to rebuild the image if you do this.